kubectl-toolbox/kubectl-toolbox

```diff
@@ -19,18 +19,20 @@ set -o pipefail  # this will make your script exit if any command in a pipeline
 ########################################################################################
 # FUNCTIONS
 
-red()    { echo -e "\033[1;31m$@\033[0m" ; }
-green()  { echo -e "\033[1;32m$@\033[0m" ; }
-yellow() { echo -e "\033[1;33m$@\033[0m" ; }
+red()    { echo -e "\033[1;31m$*\033[0m" ; }
+green()  { echo -e "\033[1;32m$*\033[0m" ; }
+yellow() { echo -e "\033[1;33m$*\033[0m" ; }
 
-ok()     { echo "$(green  "OK   - ") $1" ; }
-warn()   { echo "$(yellow "WARN - ") $1" ; }
-fail()   { echo "$(red    "FAIL - ") $1" ; }
+ok()     { echo "$(green  "OK   -") $1" ; }
+warn()   { echo "$(yellow "WARN -") $1" ; }
+fail()   { echo "$(red    "FAIL -") $1" ; }
 
 DIR=$(dirname ${BASH_SOURCE[0]})
 KUBECTL=kubectl
+KUBECTL_ARGS=""
 TOOLBOX_ENTRY_COMMAND="bash"
 TOOLBOX_NAME="toolbox-$(whoami)"
+TOOLBOX_IMAGE="ubuntu-debootstrap"
```

Comment:
SInce this now defaults to something, can you change the check in line 81 to `if default image is used, print a user message saying it.`. Something like `No docker image specified, defaulting to 'ubuntu-debootstrap' ` :)